backend/jupyterlab_metadata_service_server/src/datasources/annotation.js

```diff
@@ -92,6 +91,12 @@ class AnnotationAPI extends DataSource {
     }
   }
 
+  reducer_update_resolved(resolved) {
```

Comment:
I think for returning dictionary with just one item it is not necessary to use a reducer function.
but for now I think it is fine ... because maybe we will need make changes here when we will add a new layer for database